### ISSUE TITLE
Palettes: Color Contrast of “Cross out” icons

### ISSUE THREAD

**Issue by bpmcneilly** [benetech/MathShare#952]
Most symbols in the math palette Symbols view have more than sufficient color contrast with their backgrounds. Most icons contain a contrast ratio of over 6.0:1, far above the 3.0:1 required by WCAG. 

However, the cross out icons are unique in that in addition to the standard black icon color (#000000) these icons have a blue X indicating the cross out functionality. The X content appears in full HTML blue (#0000ff), on the black icon content for a background, this creates a color contrast ratio of 2.44:1.

#### Suggested Solution
Lightening the shade of blue (one option being #4040ff) will provide a sufficient color contrast ratio on the black icons (in the case of the suggested blue, 3.04:1).

##### Proposed Code Sample
`$$\\enclose{updiagonalstrike downdiagonalstrike}[2px solid #4040ff]{\\blacksquare}$$`
